ooniapi/services/oonirun/tests/test_database.py

```diff
@@ -176,3 +176,37 @@ def test_upgrade_to_head(postgresql):
         db.add(db_runlink)
         db.commit()
     db.rollback()
+
+def upgrade() -> None:
+    op.create_table(
+        "oonifinding",
+        sa.Column(
+            "incident_id",
+            sa.String(),
+            nullable=False,
+            primary_key=True,
+        ),
+        sa.Column("create_time", sa.DateTime(timezone=True), nullable=False),
```

Comment:
I think you are missing the `finding_slug` here and you should rename the `incident_id` to `finding_id`, also the capitalization of the `CCs` `ASNs`, etc.

Reply:
thank you for pointing this out! This was an unrelated change we had to undo since this is the oonirun service. I have removed this from the diff now.